manifests/pod.yaml

```diff
@@ -64,9 +64,11 @@ spec:
        privileged: false
      resources:
        requests:
+         cpu: COREOS_ASSEMBLER_CPU_REQUEST
          memory: COREOS_ASSEMBLER_MEMORY_REQUEST
          devices.kubevirt.io/kvm: '1'
        limits:
+         cpu: '4'
```

Comment:
We can bump the requests but I think we should leave the limit alone. If the cluster has excess CPU capacity then I don't think we should artificially limit ourselves, right?

Reply:
I was reading https://kubernetes.io/docs/tasks/administer-cluster/manage-resources/cpu-default-namespace/#what-if-you-specify-a-container-s-request-but-not-its-limit

...and was attempting to avoid a situation where the cluster/namespace defaults would arbitrarily limit the amount of CPUs we could request.  Though I cannot determine if there are default CPU limits in place.

Anyways...I'll drop the limit definition.

Reply:
yeah, since we request 4 then we won't get scheduled unless we get 4. If the cluster has a default limit then we'll get that, but it will be at least >4 since that's what we requested.